app/javascript/packages/document-capture-polling/index.js

```diff
@@ -0,0 +1,88 @@
+import { trackEvent as defaultTrackEvent } from '@18f/identity-analytics';
+
+export const DOC_CAPTURE_TIMEOUT = 1000 * 60 * 25; // 25 minutes
+export const DOC_CAPTURE_POLL_INTERVAL = 5000;
+export const MAX_DOC_CAPTURE_POLL_ATTEMPTS = Math.floor(
+  DOC_CAPTURE_TIMEOUT / DOC_CAPTURE_POLL_INTERVAL,
+);
+export const POLL_ENDPOINT = '/verify/doc_auth/link_sent/poll';
+
+/**
+ * @typedef DocumentCapturePollingElements
+ *
+ * @prop {HTMLFormElement} form
+ * @prop {HTMLParagraphElement} instructions
+ */
+
+/**
+ * @typedef DocumentCapturePollingOptions
+ *
+ * @prop {DocumentCapturePollingElements} elements
+ * @prop {typeof defaultTrackEvent=} trackEvent
+ */
+
+/**
+ * Manages polling requests for document capture hybrid flow.
+ */
+export class DocumentCapturePolling {
+  pollAttempts = 0;
+
+  /**
+   * @param {DocumentCapturePollingOptions} options
+   */
+  constructor({ elements, trackEvent = defaultTrackEvent }) {
+    this.elements = elements;
+    this.trackEvent = trackEvent;
+  }
+
+  bind() {
+    this.toggleFormVisible(false);
+    this.trackEvent('IdV: Link sent capture doc polling started');
+    this.schedulePoll();
+  }
+
+  /**
+   * @param {boolean} isVisible
+   */
+  toggleFormVisible(isVisible) {
+    this.elements.form.classList.toggle('display-none', !isVisible);
+    this.elements.instructions.classList.toggle('display-none', !isVisible);
+  }
+
+  onMaxPollAttempts() {
+    this.toggleFormVisible(true);
+  }
+
+  /**
+   * @param {{ isCancelled: boolean }} params
+   */
+  async onComplete({ isCancelled }) {
+    const event = `IdV: Link sent capture doc polling ${isCancelled ? 'cancelled' : 'complete'}`;
```

Comment:
> should this be the same event, with like a `status` attribute?

That's a good question. I think it could be. Personally, I've been a bit unclear how much we care to optimize for keying by event name vs. event properties. #4656 comes to mind here for a similar example. If I recall correctly, the thinking there was that keying by property would be generally preferable, though for consistency amongst all existing events, making a distinct event name was the best compromise.

Since we don't have to worry about such consistency here, maybe property is a better option.

cc @amathews-fs

Reply:
I think given how this will be used giving this a single named event and differentiating by property is appropriate. We can always adjust later if we want to.

Reply:
👍  Passing `isCancelled` as property instead in 7654140bb.